common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,587 @@
+		heavy_infantry = 2
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = venomancer
+}
+
+webspinner = {
+	type = archers
+	
+	damage = 35
+	toughness = 20
+	pursuit = 35
+	screen = 20
+	
+	terrain_bonus = {
+		mountains = { damage = 12 toughness = 10 }
+		taiga = { damage = 12 toughness = 10 }
+	}
+
+	counters = {
+		skirmishers = 1
+		light_aerial = 1
+	}
+
+	buy_cost = { gold = cost_105_recruitment_cost }
+	low_maintenance_cost = { gold = cost_105_low_maint_cost }
+	high_maintenance_cost = { gold = cost_105_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = webspinner
+}
+#Mantid
+kunchong = { 
+	type = massive_infantry
+	
+	damage = 131
+	toughness = 125
+	pursuit = 15
+	screen = 20
+	
+	terrain_bonus = {
+		steppe = { damage = 10 toughness = 10 pursuit = 10 }
+		mountains = { damage = 10 toughness = 10 pursuit = 10 }
+		plains = { damage = 5 toughness = 5 pursuit = 5 }
+	}
+
+	counters = {
+		heavy_cavalry = 1
+		skimishers = 3
+	}
+
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = kunchong
+}
+
+#Ogre
+mauler = {
+	type = heavy_infantry
+	
+	damage = 37
+	toughness = 40
+	pursuit = 0
+	screen = 20
+
+	counters = {
+		skirmishers = 1
+		pikemen = 1
+	}
+	
+	buy_cost = { gold = cost_108_recruitment_cost }
+	low_maintenance_cost = { gold = cost_108_low_maint_cost }
+	high_maintenance_cost = { gold = cost_108_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = mauler
+}
+
+#Pandaren
+mushan_rider = {
+	type = heavy_cavalry
+	
+	damage = 185
+	toughness = 150
+	pursuit = 60
+	screen = 25
+	
+	terrain_bonus = {
+		plains = { damage = 20 toughness = 15 pursuit = 10 }
+		drylands = { damage = 20 toughness = 15 pursuit = 10 }
+		desert = { damage = 20 toughness = 15 pursuit = 10 }
+		hills = { damage = -5 }
+		jungle = { damage = 5 }
+		mountains = { damage = -10 }
+		desert_mountains = { damage = -10 }
+		wetlands = { damage = -10 }
+	}
+
+	counters = {
+		archers = 2
+		caster = 1
+	}
+
+	buy_cost = { gold = cost_240_recruitment_cost }
+	low_maintenance_cost = { gold = cost_240_low_maint_cost }
+	high_maintenance_cost = { gold = cost_240_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = culture_ai_weight_mushan }
+	icon = mushan
+}
+#Qiriaji
+silithid_colossus = { 
+	type = massive_infantry
+	
+	damage = 135
+	toughness = 125
+	pursuit = 15
+	screen = 20
+	
+	terrain_bonus = {
+		desert = { damage = 10 toughness = 10 pursuit = 15 }
+		desert_mountains = { damage = 10 toughness = 20 pursuit = 10 }
+	}
+
+	counters = {
+		skimishers = 3
```

Comment:
I'd decrease this by 1 and add counter vs archers, because their chitineous outside would probably work well against arrows etc. And maybe increase the screen a little more as well.